widgetry/src/backend_glow_native.rs

```diff
@@ -12,16 +12,22 @@ pub fn setup(window_title: &str) -> (PrerenderInnards, winit::event_loop::EventL
     // TODO If people are hitting problems with context not matching what their GPU provides, dig up
     // backend_glium.rs from git and bring the fallback behavior here. (Ideally, there'd be
     // something in glutin to directly express this.) multisampling: 2 looks bad, 4 looks fine
-    let context = match glutin::ContextBuilder::new()
+    let context = glutin::ContextBuilder::new()
         .with_multisampling(4)
         .with_depth_buffer(2)
-        .build_windowed(window, &event_loop)
-    {
-        Ok(ctx) => ctx,
-        Err(err) => {
-            panic!("Your videocard doesn't support the OpenGL mode requested. This is a common issue when running inside a virtual machine; please run natively if possible. See https://github.com/dabreegster/abstreet/issues/103 for more info, and feel free to ask for help using that issue.\n\nError: {}", err);
-        }
-    };
+        .build_windowed(window.clone(), &event_loop).map_err(|err| {
+            warn!("Trying default graphics context after standard graphics context failed with error: {:?}",  err);
```

Comment:
Which specific cases benefit from this? Or is it just a general catch all that tries to to use default config instead?

Reply:
Yes, just a general catch-all, seemingly this wouldn't help you in particular, but the next vsync one should.